packages/meditrak-app/app/sync/Synchroniser.js

```diff
@@ -309,19 +302,30 @@ export class Synchroniser {
    * Returns the number of changes left to pull
    * @return {Promise} Resolves with the change count, or passes up any error thrown
    */
-  async getIncomingChangeCount(since, filters = {}) {
-    try {
-      const responseJson = await this.api.get(`${API_ENDPOINT}/count`, { since, ...filters });
-      if (responseJson.error && responseJson.error.length > 0) {
-        throw new Error(responseJson.error);
-      }
-      if (typeof responseJson.changeCount !== 'number') {
-        throw new Error('Unexpected response from server');
-      }
-      return responseJson.changeCount;
-    } catch (error) {
-      throw error; // Throw error up
+  async getIncomingChangeMetadata(since, filters = {}) {
+    const countriesSynced = this.database.getSetting(COUNTRIES_SYNCED);
```

Comment:
Would be nice somewhere around here to add a comment briefly describing the way sync is filtered by permissions, so the reader doesn't need to look into the server code to understand what's going on

Reply:
Great point! Will do 😄